cocotb/drivers/__init__.py

```diff
@@ -96,6 +102,7 @@ def __init__(self):
         self._thread = cocotb.scheduler.add(self._send_thread())
 
     def kill(self):
+        """Kill the coroutine sending stuff."""
```

Comment:
Maybe replace "stuff" with something more formal and specific?

Reply:
The "stuff" is actually taken from a nearby comment. I don't feel I have the knowledge to make this better without potentially introducing inaccuracies.